storybook/preview.tsx

```diff
@@ -1,26 +1,14 @@
 /* eslint import/no-extraneous-dependencies: 0 */
 import React from "react"
-import {
-  defaultTheme,
-  heartTheme,
-  ThemeProvider,
-  zenTheme,
-} from "@kaizen/design-tokens"
 import { addParameters } from "@storybook/react"
 import { addons } from "@storybook/addons"
+import KaizenContainer from "@kaizen/bootstrap"
 import { backgrounds } from "./backgrounds"
 import { themeManager, themeOfKey } from "./theme-switcher-addon/themeManager"
```

Comment:
How are we going to share a `themeManager` between our KaizenProvider and the theme-switcher-addon? We could have the addon use the exported instance in `Wrapper.tsx`, but it does get messy

Reply:
I agree that it is getting messy. 

We might want to rethink the responsibilities of the design-tokens package. Having theme management intertwined with the tokens means consumers need to use the design tokens directly to bootstrap their app. This isn't really compatible with the wrapper - we'd be exporting the ThemeManager and telling consumers _not_ to use it, and instead use the ThemeManager from Container only to avoid multiple declarations.

My current thoughts are the design tokens _should_ be for token definitions and compilations into different formats only (such as json to less/sass) and to provide these in their raw format (i.e no sass or React helpers) With that in mind, it makes sense to move the theme management responsibilities to the Container package. 

So we'd end up with something like this....

```
import KaizenContainer, { themeManager } from "@kaizen/container"
import App from "./App"

export const MyCoolApp = {
   themeSwitcher({ themeManager })

   return (
      <KaizenContainer>
        <App />
      </KaizenContainer>
   )
}
```

Then the implementation of ThemeSwitcher can be updated to consume from kaizen/container. 

WDYT?